schemas/UserProperties.json

```diff
@@ -59,6 +59,21 @@
           {"const": "Group", "description": "Modern layout."},
           {"const": "Compact", "description": "Modern layout with compact option enabled."}
         ]
+      },
+      "verificationState": {
```

Comment:
Can you add a description please? Can be copy paste of what's in CryptoSessionState.json.